### PR TITLE
Added test for Apply button

### PR DESCRIPTION
**Pull request by ataata107**
Concerns #1369  (<=== Replace `0000` with the Issue Number)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [X] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm run test-all`
* [X] code is in uniquely-named feature branch and has no merge conflicts
* [X] PR is descriptively titled
* [X] ask `@publiclab/is-reviewers` for help, in a comment below
* [X] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

**Comment by ataata107** [comment]
@publiclab/is-reviewers

File: test/ui-2/test/apply-button.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const timeout = process.env.SLOWMO ? 30000 : 10000;
+const fs = require('fs');
+beforeAll(async () => {
+  path = fs.realpathSync('file://../examples/index.html');
+  await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
+});
+function delay(time) {
```
Comment (blurry-x-face) [inline test/ui-2/test/apply-button.test.js]: Is this used anywhere? If not then I think we can remove this

File: test/ui-2/test/apply-button.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  test('Apply Button is clicked', async () => {
+    await page.waitForSelector('.step');
+
+  
+    await page.click('[data-value=\'resize\']');
+    const Length1 = await page.evaluate(() => document.querySelectorAll('.step').length);
+    //Lets change the default value
+    expect(Length1).toBe(2);
+    // Let's check the source of the image output by the default values
+    const src1 = await page.evaluate(() => document.querySelectorAll('.step img')[1].src);
```
Comment (HarshKhandeparkar) [inline test/ui-2/test/apply-button.test.js:18]: Do we need to check for all of these every time?

Comment (jywarren) [reply test/ui-2/test/apply-button.test.js:18]: I think a bit of extra testing is ok for the UI tests we badly need; we can do another phase of refining and optimizing them later but it doesn't add much overhead here! But thanks for thinking of this!

**Comment by jywarren** [comment]
Thank you for these tests! Much appreciated!!!

**Pull request merged**